api/src/main/java/io/opencensus/stats/MeasureMap.java

```diff
@@ -51,6 +52,15 @@
    */
   public abstract MeasureMap put(MeasureLong measure, long value);
 
+  /**
+   * Associate measurements with a {@link SpanContext}.
+   *
+   * @param spanContext the {@link SpanContext}
+   * @return this
+   * @since 0.15
+   */
+  public abstract MeasureMap withSpanContext(SpanContext spanContext);
```

Comment:
Can you give more details of what this is actually supposed to do (in the pr description).

Reply:
This is supposed to record a SpanContext when recording measurements. Please refer to the Specs PR: https://github.com/census-instrumentation/opencensus-specs/pull/122.